libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx

```diff
@@ -45,18 +56,18 @@ function HomeTabFeatured() {
               </div>
               <div className="mx-1 px-1 d-flex">
                 <img src={"/assets/img/remixRewardUser.webp"} style={{ flex: "1", height: "170px", maxWidth: "170px" }} alt="" ></img>
-                <div className="h6 p-4" style={{ flex: "1" }}>
+                <div className="h6 w-50 p-4" style={{ flex: "1" }}>
                   <h5><FormattedMessage id='home.remixRewards' /></h5>
-                  <p style={{ fontStyle: 'italic' }}><FormattedMessage id='home.remixRewardsText1' /></p>
+                  {/* <p style={{ fontStyle: 'italic' }}><FormattedMessage id='home.remixRewardsText1' /></p> */}
                   <p><FormattedMessage id='home.remixRewardsText2' /></p>
                   <a className="remixui_home_text" target="__blank" onClick={() => _paq.push(['trackEvent', 'hometab', 'featuredSection', 'remixRewards'])} href="https://rewards.remix.ethereum.eth.limo"><FormattedMessage id='home.more' /></a>
                 </div>
               </div>
               <div className="mx-1 px-1 d-flex">
                 <img src={"/assets/img/remixRewardBetaTester.webp"} style={{ flex: "1", height: "170px", maxWidth: "170px" }} alt="" ></img>
-                <div className="h6 p-4" style={{ flex: "1" }}>
+                <div className="h6 w-50 p-4" style={{ flex: "1" }}>
                   <h5><FormattedMessage id='home.betaTesting' /></h5>
-                  <p style={{ fontStyle: 'italic' }}><FormattedMessage id='home.betaTestingText1' /></p>
+                  {/* <p style={{ fontStyle: 'italic' }}><FormattedMessage id='home.betaTestingText1' /></p> */}
```

Comment:
why are you removing those?

Reply:
@Aniket-Engg removed those.

Reply:
links are not visible if we keep them. Also, they are not in `Jump into web3` slide. We can bring them back once UI is fixed/redesigned.

Reply:
change the font size or spaceing

Reply:
or move links to the same line